blog/index.html

```diff
@@ -0,0 +1,69 @@
+---
+title: Tilt Blog
```

Comment:
fyi -- this file is mostly unchanged -- the pagination plugin just needs the file to be called index.html